ddtrace/span.py

```diff
@@ -474,7 +474,7 @@ def set_exc_info(self, exc_type, exc_val, exc_tb):
         # readable version of type (e.g. exceptions.ZeroDivisionError)
         exc_type_str = "%s.%s" % (exc_type.__module__, exc_type.__name__)
 
-        self.meta[ERROR_MSG] = str(exc_val)
+        self.meta[ERROR_MSG] = ensure_text(exc_val.message) if six.PY2 else stringify(exc_val)
```

Comment:
```suggestion
        self.meta[ERROR_MSG] = ensure_text(exc_val.args[0] if exc_val.args else "") if six.PY2 else stringify(exc_val)
```
Ugly but should work. @brettlangdon @Kyle-Verhoog WDYT?

Reply:
same errors as before it looks like :-/

Reply:
That's weird. The message should definitely be there! 😕